zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -0,0 +1,245 @@
+    ZIP: XXX
+    Title: Deferred Dev Fund Lockbox Disbursement
+    Owners: Daira-Emma Hopwood <daira@jacaranda.org>
+            Kris Nuttycombe <kris@nutty.land>
+            Jack Grigg <jack@electriccoin.co>
+    Status: Draft
+    Category: Consensus / Process
+    Created: 2025-02-19
+    License: MIT
+    Pull-Request: <https://github.com/zcash/zips/pull/???>
+
+# Terminology
+
+The key words "MUST", "SHOULD", "MAY", and "RECOMMENDED" in this document are
+to be interpreted as described in BCP 14 [^BCP14] when, and only when, they
+appear in all capitals.
+
+# Abstract
+
+This ZIP proposes an extension of protocol-based development funding, in the
+context of multiple alternatives for distributing funds that have accrued to
+the Deferred Dev Fund Lockbox. This proposal is intended to be evaluated in the
+context of the Community And Coinholder Funding Model
+[^draft-ecc-community-and-coinholder] and Zcash Governance Bloc
+[^draft-ecc-zbloc] proposals; the mechanisms it describes are applicable to
+both of these and may be applicable to other similar proposals as well.
+
+At a high level, this ZIP proposes:
+* A one-time disbursement of the full contents of the lockbox to a transparent
+  P2SH multisig address, at the time of the activation of this ZIP. The
+  key-holders for that address are then responsible for distributing the
+  resulting funds in the form of development grants, according to the rules set
+  by either [^draft-ecc-community-and-coinholder] or [^draft-ecc-zbloc], or
+  another similar proposal.
+* Extension of protocol-based development funding blocks starting from the
+  scheduled end height of the current ``FS_DEFERRED`` and ``FS_FPF_ZCG``
+  funding streams defined in ZIP 1015 [^zip-1015-funding-streams].
+* A variety of different mechanisms that may be used for distributing funds
+  that accrue during the period of the extension, which may vary depending upon
+  which proposal this mechanism is used.
```

Comment:
"the proposal that uses this mechanism".

Reply:
Addressed in https://github.com/zcash/zips/pull/994